packages/devextreme/testing/tests/DevExpress.ui.widgets/diagramParts/options.tests.js

```diff
@@ -15,8 +45,123 @@ const moduleConfig = {
     }
 };
 
-QUnit.module('Options', {
+QUnit.module('Options: flaky', {
```

Comment:
I suggers remove `flaky` test from module name and use this this diagramm setting only for `shadow-dom` mode